package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nerdwallet/shepherd",
-  "version": "1.16.0",
+  "version": "2.0.0",
```

Comment:
This manual bump is unnecessary as semantic-release will bump the major version.